apps/bot/src/date/getFirstDateOfMonth.ts

```diff
@@ -0,0 +1,5 @@
+export const getFirstDateOfMonth = (date: Date): Date => new Date(
```

Comment:
Prefer using a library

https://momentjscom.readthedocs.io/en/latest/moment/03-manipulating/03-start-of/

Reply:
nice, I'll take a look

Reply:
I think we should use luxon btw, it's the evolution of moment

https://moment.github.io/luxon/#/?id=luxon

Reply:
I'll merge this PR and open another to use luxon.